packages/sdk/src/EventSenderEngine.ts

```diff
@@ -79,11 +79,11 @@ export class EventSenderEngine {
       .build(fetchImplementation);
   }
 
-  send(context: Value.Struct, name: string, message?: Value.Struct): void {
+  send(context: Value.Struct, name: string, data?: Value.Struct): void {
```

Comment:
There are more places where we should change "message" to "data". See EventSender interface and the Confidence class implementing it.